src/govuk/components/exit-this-page/_index.scss

```diff
@@ -66,6 +67,30 @@
     }
   }
 
+  .govuk-exit-this-page__help {
+    @include govuk-responsive-padding(2);
+    @include govuk-font($size: 16);
+    display: none;
+    border: 1px solid $govuk-border-colour;
+    border-top: none;
+    background-color: govuk-colour("white");
+    text-align: center;
+
+    kbd {
+      @include govuk-font($size: 16);
+      display: inline-block;
```

Comment:
Is this `inline-block` necessary? From a very quick VO test the screen reader is splitting the announcement of the instructions into 3 parts:

- press
- Shift
- 3 times to exit this page

...which is coming from us sort of blocking the element. Removing this makes VO interpret the line as a single announcement. The result is that visually the padding get's a little squashed which isn't a dealbreaker for me personally.

This isn't a showstopper though so happy to negotiate.

Reply:
The `inline-block` isn't needed, no. I think I added it with the expectation there'd be some vertical padding being applied, but I didn't end up using any in the end.